src/events/loginEvents.js

```diff
@@ -1,22 +1,29 @@
-const AddEventsLogin = function() {
-    const listener = function(e) {
+interview.AddEventsLogin = function () {
```

Comment:
Is this truly needed as a publicly exposed method?

Reply:
well, that's how I managed to do so it doesn't appear on window... Is this ok?